blog/2018-07-19-advantages-of-log-based-cdc.adoc

```diff
@@ -0,0 +1,66 @@
+= Five Advantages of Log-Based CDC
+gmorling
+:awestruct-tags: [ discussion ]
+:awestruct-layout: blog-post
+
+Yesterday I had the opportunity to present Debezium and the idea of change data capture to the https://twitter.com/JUG_DA/status/1019634941020332032[Darmstadt Java User Group].
+It was a great evening with lots of interesting discussions and questions.
+One of the questions being the following: what is the advantage of using a log-based change data capturing tool such as Debezium over simply polling for updated records e.g. via JDBC?
+
+I replied with a couple of advantages of using Debezium.
+But as this wasn't the first time this question came up, I thought I could provide a more extensive answer here on the blog.
+That way I'll be able to refer to this post in the future, should the question come up again :)
+
+So without further ado, here's my list of five advantages of log-based CDC over polling-based (or query-based) approaches.
+
+All Data Changes Are Captured:: By reading the database's log (e.g. MySQL's binlog or MongoDB's op log) you get the complete list of all data changes in their exact order of application.
+This is vital for many use cases where you are interested in the complete history of record changes.
```

Comment:
Maybe it might be usefule to emphasize that you are able to reconstruct the database state at arbitrary point of time.